sdk/src/main/scala/io/horizen/utxo/SidechainApp.scala

```diff
@@ -214,6 +213,8 @@ class SidechainApp @Inject()
 
   val boxIterator: BoxIterator = backupStorage.getBoxIterator
 
+  override lazy val applicationApiRoutes: Seq[SidechainApplicationApiRoute] = customApiGroups.asScala.map(apiRoute => SidechainApplicationApiRoute(settings.restApi, apiRoute, nodeViewHolderRef))
```

Comment:
```suggestion
  override lazy val applicationApiRoutes: Seq[ApiRoute] = customApiGroups.asScala.map(apiRoute => SidechainApplicationApiRoute(settings.restApi, apiRoute, nodeViewHolderRef))
```
same as in Account class

Reply:
Done